data/init/dfhack.keybindings.init

```diff
@@ -25,6 +25,9 @@ keybinding add Ctrl-Shift-K gui/cp437-table
 # customizable quick command list
 keybinding add Ctrl-Shift-A gui/quickcmd
 
+# save the description of a selected unit or item to the `/markdown_{YourWorldName}.md` file in the root game directory
```

Comment:
starting a filename with a slash indicates that it will be saved in the root folder of the entire system, not the current directory. I'd remove that leading slash.